code/game/objects/items/food/pastries.dm

```diff
@@ -637,7 +637,7 @@
 /obj/item/food/cookie/sugar
 	name = "sugar cookie"
 	desc = "Just like your little sister used to make."
-	icon_state = "sugarcookie"
+	icon_state = "sugarcookie_pbird"
```

Comment:
Maybe just make a subtype of sugar cookies?

Reply:
Seems a bit redundant though.